sdk/playwrighttesting/microsoft-playwright-testing/src/reporter/mptReporter.ts

```diff
@@ -51,6 +51,7 @@ class MPTReporter implements Reporter {
   private isTokenValid: boolean = true;
   private enableGitHubSummary = true;
   private isRegionValid: boolean = true;
+  private isSameWorkspace: boolean = true;
```

Comment:
Do not introduce another variable, you can utilize isTokenValid since this is a scenario when the token is not valid(belongs to a different workspace)